onnx/compose.py

```diff
@@ -0,0 +1,313 @@
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from typing import List, Tuple, Text, Optional, MutableMapping
+from onnx import ModelProto, GraphProto, helper, checker
+from onnx import TensorProto as tp
+
+
+def merge_graphs(
```

Comment:
What about name clashes between the two graphs? Eg., if they both use some temporary with the same name? Conceptually, it should be straight-forward to make the names distinct by adding some prefix/suffix in this case. Right now, it looks like this is not handled?

Reply:
I see a separate utility is added later for this. May be we could have some higher-level function that calls both (the renamer and merge_graphs)?

Reply:
I've addressed that. Now both merge_graphs or merge_models accept optional prefix1 and prefix2 arguments, that if supplied will be used to rename all relevant names in each model/graph